spec/functional/win32/version_info_spec.rb

```diff
@@ -33,11 +33,11 @@
   subject { Chef::ReservedNames::Win32::File::VersionInfo.new(file_path) }
 
   it "file version has the same version as windows" do
-    expect(subject.FileVersion).to start_with(os_version)
+    expect(subject.FileVersion).to start_with(os_version.rpartition('.').first)
```

Comment:
can you check the it to mention that we're trying to match major/minor only here now